### ISSUE TITLE
engines field is too strict in 3.2.0

### ISSUE THREAD

**Issue by twhid** [nfl/react-helmet#187]
Can't install with node@lts, which is 6.9.1.

```
$ yarn
yarn install v0.16.1
info No lockfile found.
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error react-helmet@3.2.0: The engine "node" is incompatible with this module. Expected version "6.6.0".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

```

**Comment by cwelch5** [comment]
Sorry, will fix shortly.

**Comment by twhid** [comment]
No worries, sorry to be so terse! Easy to workaround 😄

**Comment by cwelch5** [comment]
No problem.  Thanks for the heads up!  This is now fixed in v3.2.1.
